packages/ocap-kernel/src/VatSyscall.ts

```diff
@@ -63,8 +63,7 @@ export class VatSyscall {
     this.vatId = vatId;
     this.#kernelQueue = kernelQueue;
     this.#kernelStore = kernelStore;
-    this.#logger =
-      logger ?? new Logger({ tags: [`[vat ${vatId}]`, 'syscall'] });
+    this.#logger = logger;
```

Comment:
```suggestion
    this.#logger =
      logger ?? new Logger({ tags: [`[vat ${vatId}]`, 'syscall'] });
```

I liked the previous code and that Logger was defined always.

Reply:
https://github.com/MetaMask/ocap-kernel/pull/559#discussion_r2183616701 

Having the subcomponents introduce themselves to the logger (even if only out of laziness of their parents) seems ... inadvisable. In conflict with reuse.